pkg/apis/aws/validation/shoot.go

```diff
@@ -91,6 +91,27 @@ func ValidateWorkersUpdate(oldWorkers, newWorkers []core.Worker, fldPath *field.
 	return allErrs
 }
 
+func ValidateNetworkingUpdate(oldNetworkConfig, networkConfig *core.Networking, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, ValidateIPFamiliesUpdate(oldNetworkConfig.IPFamilies, networkConfig.IPFamilies, fldPath.Child("ipFamilies"))...)
+	return allErrs
```

Comment:
Nit: You could make this a single line, but for the sake of consistency it could also be left as is. I see other functions below, which just have a single line check.